types/react-click-outside-hook/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for react-click-outside-hook 1.0
+// Project: https://github.com/bdeloeste/react-click-outside-hook#readme
+// Definitions by: Bryan Deloeste <https://github.com/bdeloeste>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+import { MutableRefObject } from "react";
+
+export function useClickOutside<T>(): [MutableRefObject<T>, boolean];
```

Comment:
CI seems to be failing because of this. Any suggestions on how to better handle the `no-unnecessary-generics` rule here? I'd like to return a more descriptive type by passing the generic type to the `MutableRefObject`.